internal/deviations/deviations.go

```diff
@@ -222,4 +222,10 @@ var (
 
 	ISISRestartSuppressUnsupported = flag.Bool("deviation_isis_restart_suppress_unsupported", false,
 		"Device skip isis restart-suppress check if value is true, Default value is false")
+
+	ISISGlobalAuthenticationNotRequired = flag.Bool("deviation_isis_global_authentication_not_required", false,
```

Comment:
Please follow the new style for any new deviations. The flag should be unexported and accessed through an exported accessor method that accepts a DUT, e.g.
https://github.com/openconfig/featureprofiles/blob/main/internal/deviations/deviations.go#L73
This is going to enable our transition away from flags to reading vendor-specific deviations from the metadata textproto.

Sample PR to see the deviation and use in test - https://github.com/openconfig/featureprofiles/pull/1358/files

Reply:
Sure. I will make the changes accordingly and push the commit.

Reply:
Updated deviations as per new style.